### PR TITLE
feat: implement base ecosystem

### PR DESCRIPTION
**Pull request by bilbeyt**
### What I did

fixes: #

### Checklist

- [ ] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [ ] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Quick Start
 
-TODO: Description
+Ecosystem Plugin for Base support in Ape.
```
Comment (antazoey) [inline README.md:3]: wouldnt hurt to link to base here

File: ape_base/ecosystem.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                r=bytes(kwargs["r"]),
+                s=bytes(kwargs["s"]),
+            )
+
+        return txn_class.parse_obj(kwargs)
+
+
+def _get_transaction_type(_type: Optional[Union[int, str, bytes]]) -> TransactionType:
+    if not _type or _type == 126:
+        return TransactionType.STATIC
```
Comment (antazoey) [inline ape_base/ecosystem.py:94]: same deal - they don't do eip 1559 yet?

**Review by antazoey — APPROVED** [review]
looks good!

File: ape_base/ecosystem.py (hunk 2 above)
Comment (bilbeyt) [reply ape_base/ecosystem.py:94]: it is basically using OP stack and OP is not using that yet

File: README.md (hunk 1 above)
Comment (bilbeyt) [reply README.md]: done

**Comment by bilbeyt** [comment]
@antazoey if this is ok like this, i would like to merge and i should transfer this to apeworx right ?

**Pull request merged**